Sources/App/Core/DocumentationTarget.swift

```diff
@@ -17,7 +17,12 @@ import Fluent
 
 enum DocumentationTarget: Equatable, Codable {
     case external(url: String)
-    case `internal`(reference: String, archive: String)
+    case `internal`(reference: Reference, archive: String)
+
+    enum Reference: Equatable, Codable {
+        case current
+        case reference(String)
+    }
```

Comment:
This is the exact same type as `DocRoute.DocVersion`. I'll have a look if they can be merged, it feels weird to have this declared twice.